members.py

```diff
@@ -35,5 +44,14 @@ def add_member(data):
     cn.commit()
 
 def delete_member(member_id):
-    cn.execute('DELETE FROM members WHERE ROWID = ?', member_id)
+    cn.execute('UPDATE members SET enabled = 0 WHERE id = ?', member_id)
```

Comment:
Here's an example of that two-param .execute() method I was talking about. If you use this one in main.py, we should be good :)

Reply:
Hmm, ok so the commit I just did automatically updated the other one, but the note you left here seems to still flag it as having a change requested.  What do we need to do about this one?

Reply:
That just means no one had reviewed the changes yet. I just reviewed (and approved). This branch is ready to be merged in now, which I'll do in just a moment.